runtime/compiler/z/codegen/J9S390PrivateLinkage.cpp

```diff
@@ -200,7 +200,6 @@ TR::S390PrivateLinkage::initS390RealRegisterLinkage()
       freeingSSPDisabled = false;
 
    if (freeingSSPDisabled || disableFreeJITSSP != NULL ||
-       !(cg()->supportsJITFreeSystemStackPointer()) ||
```

Comment:
Was the removal of `!(cg()->supportsJITFreeSystemStackPointer())` an intended change in this commit?

Reply:
Yes it was. If you look at a few lines above you see the following:

```
   bool freeingSSPDisabled = true;

   TR_J9VMBase *fej9 = (TR_J9VMBase *)(fe());

   if (cg()->supportsJITFreeSystemStackPointer())
      freeingSSPDisabled = false;
```

Thus `freeingSSPDisabled == !cg()->supportsJITFreeSystemStackPointer()` and `freeingSSPDisabled` is already part of the OR chain of the if statement on line 202 so querying `!(cg()->supportsJITFreeSystemStackPointer())` is redundant and hence can be removed.